### PR TITLE
FIX #27 Added XML Layout for Forgot Password Activity

### LINKED ISSUE #27: XML layout for Forgot Password Activity
Create the XML layout for Forgot Password Activity based on the design provided in the [Figma](https://www.figma.com/design/jYm7XGidk1L3HNf9e4Otwn/myLeave?node-id=6-18&t=HjrOtyAfoLbxQDPn-1).

Your layout must follow these guidelines:

1. Use Constraint Layout to ensure the layout is responsive to all screen sizes.
2. Try to maintain uniformity with the Sign Up and Login page, you can reuse the code.
3. Adhere to the given Figma design
4. Follow the specified naming convention for each view group ID to maintain consistency and clarity.

### PR DESCRIPTION
**Pull request by aniket0742**
## Overview
1. This PR fixes or fixes part of #27.
2. This PR does the following: XML Layout for the Forgot Password page.

## Essential Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] "Allow edits from maintainers" is checked. (See [here](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) for instructions on how to enable it.)
- [x] The PR is made from a branch that's **not** called "main/master".

## Proof that changes are correct
Screenshot 2024-07-06 220441

## PR Pointers

- If you need a review or an answer to a question, and don't have permissions to assign people, **leave a comment** like the following: "{{Question/comment}} @{{reviewer_username}} PTAL".
- Never force push. If you do, your PR will be closed.

### CODE REVIEW

File: app/src/main/res/layout/activity_forgot_password.xml [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:cardBackgroundColor="@color/sign_up_image"
+        android:elevation="4dp"
+        android:layout_marginBottom="-24dp">
+
+
+        <ImageView
+            android:id="@+id/tv_forgot_passwordImg"
```
Comment (Agrim2112) [inline app/src/main/res/layout/activity_forgot_password.xml:26]: Change tv to iv

File: app/src/main/res/layout/activity_forgot_password.xml [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                        android:fontFamily="@font/inter_regular"
+                        android:hint="@string/hint_email"
+                        android:inputType="textEmailAddress"
+                        android:padding="12dp"
+                        android:textColor="@color/black"
+                        android:textColorHint="@color/leaves_txt"
+                        android:textSize="16sp" />
+
+                    <TextView
+                        android:id="@+id/space"
```
Comment (Agrim2112) [inline app/src/main/res/layout/activity_forgot_password.xml:126]: Remove this empty Text View

File: app/src/main/res/layout/activity_forgot_password.xml [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginBottom="144dp"
+                    android:backgroundTint="@color/btn_color"
+                    android:fontFamily="@font/lato_bold"
+                    android:paddingVertical="2dp"
+                    android:text="@string/reset"
+                    android:textColor="@color/white"
+                    android:textSize="20sp"
+                    app:layout_constraintBottom_toTopOf="@+id/llLoginLink"
```
Comment (Agrim2112) [inline app/src/main/res/layout/activity_forgot_password.xml:144]: Change it to Top_to_Bottomof

File: app/src/main/res/layout/activity_forgot_password.xml [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="24dp"
+                        />
+                </LinearLayout>
+
+                <Button
+                    android:id="@+id/btnReset"
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginBottom="144dp"
```
Comment (Agrim2112) [inline app/src/main/res/layout/activity_forgot_password.xml:137]: Remove bottom margin and instead add appropriate top margin to adjust the button position

**Review by Agrim2112 — COMMENTED** [review]
Make the requested changes

**Comment by aniket0742** [comment]
made the required changes.   
Screenshot 2024-07-12 140828

**Pull request merged**